server/py/framework/utils/clients/messaging.py

```diff
@@ -282,3 +289,20 @@ async def _resolve_request_kwargs_from_request(
 
         request_kwargs.update(**kwargs)
         return request_kwargs
+
+    @staticmethod
+    def _get_prefix_and_version(path: str):
+        match = PREFIX_GROUPING.match(path)
+        prefix = match.group(1)
+        version = match.group(2) or "v1"
+        return prefix, version
```

Comment:
if doesn't match, match will be None, as a result `match.group(1)` will raise an error. is it expected?